org.eclipse.xtend.lib/build.gradle

```diff
@@ -6,3 +6,23 @@ dependencies {
 	compile project(':org.eclipse.xtext.xbase.lib')
 	compile project(':org.eclipse.xtend.lib.macro')
 }
+
+task copyGwtSources(dependsOn: 'build') {
```

Comment:
This task should not be here, it belongs in the xtend.lib.gwt project. That project should have a Configuration that includes source jars and unpacks those for inclusion into its own jar. 

Also there is no reason for this task to depend on 'build'. That would mean you run all tests and other checks just to build a source jar.